starter-project/js/src/agent.js

```diff
@@ -52,9 +52,30 @@ const handleTransaction = async (txEvent) => {
 //   return findings;
 // };
 
+const handleAlert = async (alertEvent) => {
+  const findings = [];
+  console.log(JSON.stringify(alertEvent))
+  if (alertEvent.alert.name === "FORTA-1"){
+      // combine some alerts
+      findings.push(
+          Finding.fromObject({
+              name: "High Tether Transfer Alert Detected",
+              description: `High amount of USDT transferred`,
+              alertId: "FORTA-2",
+              severity: FindingSeverity.Low,
+              type: FindingType.Info,
+              metadata: {},
+          })
+      );
+  }
+
+  return findings;
+};
+
```

Comment:
i might declare the `handleAlert` function (so that devs are aware of it) but leave it empty and commented out similar to handleBlock. reasoning: since we are not using the initialize handler here to subscribe to specific bots, i think this could cause confusion for new developers on how to use this correctly. a more complete example might make sense in the forta-bot-examples repo

Reply:
Do you think we should build an example in the forta-bot-examples repo?